deployment/ccip/changeset/v1_5_1/cs_configure_token_pools_test.go

```diff
@@ -99,6 +101,35 @@ func validateMemberOfTokenPoolPair(
 	}
 }
 
+func validateSolanaConfig(t *testing.T, state changeset.CCIPOnChainState, solChainUpdates map[uint64]v1_5_1.SolChainUpdate, selector uint64, solanaSelector uint64) {
+	tokenPool := state.Chains[selector].BurnMintTokenPools[testhelpers.TestTokenSymbol][deployment.Version1_5_1]
+	isSupported, err := tokenPool.IsSupportedChain(nil, solanaSelector)
+	require.NoError(t, err)
```

Comment:
nit: Can we add a additional description here and below as well?
```suggestion
	require.NoError(t, err, "unsupported chain %d", solanaSelector)
```